sshdconfig/src/util.rs

```diff
@@ -36,16 +66,26 @@ pub fn enable_tracing() {
 /// # Errors
 ///
 /// This function will return an error if sshd -T fails to validate `sshd_config`.
-pub fn invoke_sshd_config_validation() -> Result<String, SshdConfigError> {
+pub fn invoke_sshd_config_validation(args: Option<SshdCommandArgs>) -> Result<String, SshdConfigError> {
     let sshd_command = if cfg!(target_os = "windows") {
```

Comment:
I don't think on Windows you MUST have the `.exe` extension, so it might be sufficient to not have this code and always use `sshd`